.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: "Stale label"
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  branches:
+    - dev-es    
+jobs:
+  stale:
+    timeout-minutes: 1
```

Comment:
do we need to set this parameter ?

Reply:
I think, this parameter reduce the default time, and the waiting time in the job.
Check " The maximum number of minutes to let a job run before GitHub automatically cancels it. Default: 360" in https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes